nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv

```diff
@@ -0,0 +1,71 @@
+Hon.	honorable
```

Comment:
is this necessary?

Reply:
I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

Reply:
same suggestion for all whitelist files that contain words in English